dtformats/unified_logging.py

```diff
@@ -354,6 +358,28 @@ class TraceV3File(data_format.BinaryDataFile):
       ('compressed_data_size', 'Compressed data size',
        '_FormatIntegerAsDecimal')]
 
+  _DEBUG_INFO_OVERSIZE_CHUNK = [
+      ('process_identifier1', 'Process Identifier1', '_FormatIntegerAsDecimal'),
+      ('process_identifier2', 'Process Identifier2', '_FormatIntegerAsDecimal'),
+      ('unknown3', 'Unknown3', '_FormatIntegerAsHexadecimal8'),
+      ('continuous_time', 'Continuous Time', '_FormatIntegerAsDecimal'),
+      ('data_reference_index', 'Data Reference Index',
+       '_FormatIntegerAsDecimal'),
+      ('data_size', 'Data Size', '_FormatIntegerAsDecimal')]
+
+  _DEBUG_INFO_STATEDUMP_CHUNK = [
+      ('process_identifier1', 'Process Identifier1', '_FormatIntegerAsDecimal'),
+      ('process_identifier2', 'Process Identifier2', '_FormatIntegerAsDecimal'),
+      ('unknown3', 'Unknown3', '_FormatIntegerAsHexadecimal8'),
+      ('continuous_time', 'Continuous Time', '_FormatIntegerAsDecimal'),
+      ('activity_identifier', 'Activity Identifier', '_FormatIntegerAsDecimal'),
+      ('uuid', 'UUID', '_FormatUUIDAsString'),
+      ('data_type', 'Data Type', '_FormatIntegerAsDecimal'),
+      ('data_size', 'Data Size', '_FormatIntegerAsDecimal'),
+      ('object_type_string1', 'Object Type String1', '_FormatCharacterStream'),
```

Comment:
At first I defined object_type_string1 and object_type_string2 as strings, but some fields in my samples contained non-ascii codes.

Reply:
which encoding are these using?